test/ring/middleware/servlet_context_test.clj

```diff
@@ -0,0 +1,52 @@
+(ns ring.middleware.servlet-context-test
+  (:require [clojure.test :refer :all]
+            [ring.util.response :as response]
+            [ring.util.servlet :as servlet]
+            [ring.adapter.jetty :as jetty]
+            [clj-http.client :as http]
+            [ring.middleware.servlet-context :as sc])
+  (:import org.eclipse.jetty.server.Server
+           [org.eclipse.jetty.servlet ServletContextHandler ServletHolder]))
+
+(defn- wrap-servlet-handler [handler opts]
+  (fn [^Server server]
+    (->> "/*"
+         (.addServlet (ServletHolder. (servlet/servlet handler opts)))
+         (doto (ServletContextHandler.)
+           (.setContextPath (:servlet-context-path opts)))
+         (.setHandler server))))
+
+(defn- get-resolved-uri [{:keys [async? port servlet-context-path] :as opts}]
+  (let [handler (sc/wrap-servlet-context 
```

Comment:
Please remove a traling space here.

Reply:
Thank you for pointing that out! I amended the commit.